kloppy/domain/models/common.py

```diff
@@ -21,6 +21,20 @@ class Score:
     away: int
 
 
+@dataclass
+class Formation:
+    """
+    Formation
+
+    Attributes:
+        home:
+        away:
+    """
+
+    home: str
```

Comment:
the `home` and `away` attributes should be of type `FormationType`

Reply:
Second though. I think it would be good to move this to `Team.starting_formation`